src/components/Toastr/Toast.jsx

```diff
@@ -10,11 +10,11 @@ const Toast = ({ type, message, buttonLabel = "", onClick = noop }) => (
     data-test={`toastr-${type}-container`}
   >
     <Linkify
-      componentDecorator={(decoratedHref, decoratedText, key) => (
-        <a href={decoratedHref} key={key} rel="noreferrer" target="_blank">
-          {decoratedText}
-        </a>
-      )}
+      options={{
+        defaultProtocol: "https",
+        rel: "noreferrer",
+        target: "_blank",
+      }}
```

Comment:
Does this also consider `emails` that needs to be displayed as links wich will open a mail client on click?

Reply:
Yes. I have verified it and checked the docs too - [link](https://linkify.js.org/docs/linkify-react.html)